### PR TITLE
like button + responsive

### PR DESCRIPTION
**Pull request by soon-404**

### CODE REVIEW

File: app/src/components/common/BlogCard.tsx [hunk 1]
```diff
@@ -1,35 +1,47 @@
...
-
 type BlogCardProps = Pick<Blog, 'date' | 'exText' | 'like' | 'name' | 'photo' | 'topic'>
 
+
 export const BlogCard = ({ date, exText, like, name, photo, topic }: BlogCardProps) => {
+  const [clickLike, setClickLike] = useState<boolean>(true)
+  const [likePhoto, setLikePhoto] = useState<string>(unlikeImg)
+  const handleLike = () => {
+    setClickLike(!clickLike)
+    if (clickLike === true) {
```
Comment (swekiiz) [inline app/src/components/common/BlogCard.tsx:14]:
```ts
if(clickLike)
```

File: app/src/components/common/BlogCard.tsx [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
-      <div className="flex items-center justify-between h-36 ">
-        <div className="flex flex-col items-center justify-center w-1/6 h-full p-1 lg:p-5 rounded-2xl">
-          <img src={photo} className="w-12 h-12 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20"></img>
-          <p className="pt-2 text-sm md:text-md">{name}</p>
-        </div>
-        <div className="w-4/6 h-full p-5">
+    <div className="relative flex flex-col mb-4 h-68 md:flex-row md:h-48 md:w-3/5 w-80 lg:w-4/5 lg:h-40 bg-primary-lightest rounded-2xl">
+      {/* profile picture + name */}
+      <div className="flex flex-row items-center w-full h-full p-1 md:justify-center md:w-1/6 md:flex-col lg:p-5 rounded-2xl">
+        <img src={photo} className="w-12 h-12 mx-4 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20 md:mx-0"></img>
```
Comment (swekiiz) [inline app/src/components/common/BlogCard.tsx:26]:
```ts
<img src={photo} className="w-12 h-12 mx-4 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20 md:mx-0" />
```

File: app/src/components/common/BlogCard.tsx [hunk 3]
```diff
@@ -1,35 +1,47 @@
...
+        <p className="pt-2 text-md md:text-md">{name}</p>
+      </div>
+      <div className="flex flex-row w-full md:w-5/6">
+        {/* topic + preview */}
+        <div className="w-3/4 h-full p-5 pt-0 pr-0 md:pt-3">
           <p className="font-bold md:text-xl">{topic}</p>
-          <p className="py-2 text-sm lg:text-lg">{exText.slice(0, 200) + ' ...'}</p>
+          <p className="py-2 text-sm lg:text-lg">{exText.slice(0, 160) + ' ...'}</p>
```
Comment (swekiiz) [inline app/src/components/common/BlogCard.tsx:33]:
```ts
<p className="py-2 text-sm lg:text-lg">{`${exText.slice(0, 160)} ...`}</p>
```

File: app/src/components/common/BlogProfileCard.tsx [hunk 4]
```diff
@@ -1,38 +1,50 @@
...
-// }
-
 type BlogProfileCardProps = Pick<BlogProfile, 'date' | 'exText' | 'like' | 'name' | 'photo' | 'topic'>
 
 export const BlogProfileCard = ({ date, exText, like, name, photo, topic }: BlogProfileCardProps) => {
+  const [clickLike, setClickLike] = useState<boolean>(true)
+  const [likePhoto, setLikePhoto] = useState<string>(unlikeImg)
+  const handleLike = () => {
+    setClickLike(!clickLike)
+    if (clickLike === true) {
```
Comment (swekiiz) [inline app/src/components/common/BlogProfileCard.tsx:14]: same as above

File: app/src/components/common/BlogProfileCard.tsx [hunk 5]
```diff
@@ -1,38 +1,50 @@
...
+      <div className="flex flex-row items-center w-full h-full p-1 mr-4 md:justify-center md:w-1/6 md:flex-col lg:p-5 rounded-2xl">
+        <img src={photo} className="w-12 h-12 mx-4 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20 md:mx-0"></img>
+        <p className="pt-2 text-md md:text-md">{name}</p>
+      </div>
+      <div className="flex flex-row w-full md:w-5/6">
+        {/* topic + preview */}
+        <div className="w-3/4 h-full p-5 pt-0 pr-0 md:pt-3">
           <p className="font-bold md:text-xl">{topic}</p>
           <p className="py-2 text-sm lg:text-lg">{exText.slice(0, 160) + ' ...'}</p>
         </div>
```
Comment (swekiiz) [inline app/src/components/common/BlogProfileCard.tsx:34]: same as above

File: app/src/components/common/CommentCard.tsx [hunk 6]
```diff
@@ -1,34 +1,49 @@
...
-// }
-
 type CommentCardProps = Pick<Comment, 'date' | 'like' | 'name' | 'photo' | 'text'>
 
 export const CommentCard = ({ date, like, name, photo, text }: CommentCardProps) => {
+  const [clickLike, setClickLike] = useState<boolean>(true)
+  const [likePhoto, setLikePhoto] = useState<string>(unlikeImg)
+  const handleLike = () => {
+    setClickLike(!clickLike)
+    if (clickLike === true) {
```
Comment (swekiiz) [inline app/src/components/common/CommentCard.tsx:14]: same as above

File: app/src/components/common/CommentCard.tsx [hunk 7]
```diff
@@ -1,34 +1,49 @@
...
-          <img src={photo} className="w-20 h-20 mt-2 bg-blue-300 rounded-full"></img>
-          <p className="pt-2">{name}</p>
-        </div>
-        <div className="w-4/6 h-full p-5">
-          <p className="text-xl font-bold"></p>
-          <p className="py-2 text-md">{text}</p>
+    <div className="relative flex flex-col mb-4 h-68 md:flex-row md:h-48 md:w-3/5 w-80 lg:w-4/5 lg:h-40 bg-primary-lightest rounded-2xl">
+      {/* profile picture + name */}
+      <div className="flex flex-row items-center w-full h-full p-1 mr-4 md:justify-center md:w-1/6 md:flex-col lg:p-5 rounded-2xl">
+        <img src={photo} className="w-12 h-12 mx-4 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20 md:mx-0"></img>
```
Comment (swekiiz) [inline app/src/components/common/CommentCard.tsx:26]: same as above

File: app/src/components/common/BlogProfileCard.tsx [hunk 8]
```diff
@@ -1,38 +1,50 @@
...
-      <div className="flex items-center justify-between h-36 ">
-        <div className="flex flex-col items-center justify-center w-1/6 h-full p-1 lg:p-5 rounded-2xl">
-          <img src={photo} className="w-12 h-12 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20"></img>
-          <p className="pt-2 text-sm md:text-md">{name}</p>
-        </div>
-        <div className="w-4/6 h-full p-5">
+    <div className="relative flex flex-col mb-4 h-68 md:flex-row md:h-48 md:w-3/5 w-80 lg:w-4/5 lg:h-40 bg-primary-lightest rounded-2xl">
+      {/* profile picture + name */}
+      <div className="flex flex-row items-center w-full h-full p-1 mr-4 md:justify-center md:w-1/6 md:flex-col lg:p-5 rounded-2xl">
+        <img src={photo} className="w-12 h-12 mx-4 mt-2 bg-blue-300 rounded-full md:w-20 md:h-20 md:mx-0"></img>
```
Comment (swekiiz) [inline app/src/components/common/BlogProfileCard.tsx:26]: same as above

File: app/src/components/common/Story.tsx [hunk 9]
```diff
@@ -1,53 +1,56 @@
...
-
+import { useState } from 'react'
+import likeImg from 'assets/images/like.png'
+import unlikeImg from 'assets/images/unlike.png'
 export const Story = () => {
+  const [clickLike, setClickLike] = useState<boolean>(true)
+  const [likePhoto, setLikePhoto] = useState<string>(unlikeImg)
+  const handleLike = () => {
+    setClickLike(!clickLike)
+    if (clickLike === true) {
```
Comment (swekiiz) [inline app/src/components/common/Story.tsx:10]: same as above

File: app/src/pages/editProfile/index.tsx [hunk 10]
```diff
@@ -4,17 +4,17 @@ import { Link } from 'react-router-dom'
...
-  const [password, setPassword] = useState('')
-  const [confirmpassword, setConfirmpassword] = useState('')
-
+  const [username, setUsername] = useState<string>('')
+  const [displayname, setDisplayname] = useState<string>('')
+  const [password, setPassword] = useState<string>('')
+  const [confirmpassword, setConfirmpassword] = useState<string>('')
   const [image, setImage] = useState<string>(profile1)
   const [file, setFile] = useState<File | null>()
+  const [modal,setModal] = useState<boolean>(false)
```
Comment (swekiiz) [inline app/src/pages/editProfile/index.tsx:17]: isModalOpen, setIsModalOpen

**Review by eXitHere — APPROVED** [review]

**Pull request merged**